src/main/java/dook/Dook.java

```diff
@@ -0,0 +1,71 @@
+package dook;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import dook.commands.Command;
+import dook.parser.Parser;
+import dook.storage.Storage;
+import dook.tasks.TaskList;
+import dook.ui.Ui;
+
+public class Dook {
```

Comment:
Could it be a good idea to add JavaDoc comments as this is a non-trivial public class?